modular_skyrat/modules/syndie_silicon/code/interdyne_emag.dm

```diff
@@ -0,0 +1,8 @@
+/obj/item/card/emag/interdyne
+	name = "interdyne silicon card"
+	special_desc = "A modified ID card used to override silicon for use on the DS-2 and Interdyne stations. Do not bite."
```

Comment:
interdyne can't see this

Reply:
![image](https://user-images.githubusercontent.com/106692773/179270706-7cb73122-d103-4a4a-83d9-30701d191fec.png)
Same goes for DS-2 operatives!